### ISSUE TITLE
Listen to multiple node debug processes

### ISSUE THREAD

**Issue by keleko34** [TheBenji/brackets-node-debugger#14]
when there are forked processes multiple debugger ports are assigned to each, is it possible to watch all at the same time rather then just one?

**Comment by TheBenji** [comment]
Nope, currently not. When I built it I used it mostly for APIs so there was no need for this. Feel free to fork and build it tho ;)
